conftest.py

```diff
@@ -30,6 +30,15 @@ def __str__(self):
         return "{} (port {})".format(self.message, self.port)
 
 
+def pytest_configure(config):
+    if store.current_appliance.version < '5.6':
```

Comment:
Not a valid reason to block this PR from being merged. `store.current_appliance` is used all over the place and we need to get rid of all of them...